Add Cucumber for VSCode

mpkorstanje opened the pull request:
### ⚡️ What's your motivation? 
While the documentation mentioned the existence of VS Code, it didn't include the Cucumber for VSCode extension. A bit of an oversight.

Originally pointed out by dheerajramchandani in https://github.com/cucumber/common/discussions/2266

### 🏷️ What kind of change is this?

- :book: Documentation (improvements without changing code)

mattwynne commented on apps/dynamic/views/events/cukeup-au-2016.slim:
```diff
@@ -0,0 +1,160 @@
+nav: events
+title: CukeUp! Australia
+image: /images/blog/arti-workshop-cukeup-2015.jpg
+ical_url: 
```
Change the value of this line to `http://lanyrd.com/2016/cukeupau/` and it should start showing up at https://cucumber-website-pr-130.herokuapp.com/events

theo-england replied:
Think it works. When you click through the herokuapp it doesn't give you the right URL. It looks fine at https://cucumber-website-pr-130.herokuapp.com/events/cukeup-au-2016

mattwynne commented:
You'll need to fix he ical_url otherwise the link from /events won't work,
and then the page won't be visible to google.
On Tue, 15 Mar 2016 at 15:55, Theo <notifications@github.com> wrote:

> Merged #130 <https://github.com/cucumber/website/pull/130>.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub:
>
> https://github.com/cucumber/website/pull/130#event-590506743
>

-- 



The Cucumber logo is the intellectual property of Cucumber Ltd, a limited 
company registered in Scotland, number 456793.

UK Headquarters: Cucumber Ltd, Drumsyniebeg, Lochgoilhead, Cairndow, 
Argyll, PA24 8AN UK.

CONFIDENTIALITY NOTICE: The information in this e-mail is confidential and 
privileged; it is intended for use solely by the individual or entity named 
as the recipient hereof. Disclosure, copying, distribution, or use of the 
contents of this e-mail by persons other than the intended recipient is 
strictly prohibited and may violate applicable laws. If you have received 
this e-mail in error, please delete the original message and notify us by 
email immediately. Thank you. Cucumber Ltd.

mattwynne commented:
Ok maybe I'm taking bollocks - it's working!
On Tue, 15 Mar 2016 at 20:05, Matt Wynne <matt@cucumber.io> wrote:

> You'll need to fix he ical_url otherwise the link from /events won't work,
> and then the page won't be visible to google.
> On Tue, 15 Mar 2016 at 15:55, Theo <notifications@github.com> wrote:
>
>> Merged #130 <https://github.com/cucumber/website/pull/130>.
>>
>> —
>> You are receiving this because you commented.
>>
>>
>> Reply to this email directly or view it on GitHub:
>>
>> https://github.com/cucumber/website/pull/130#event-590506743
>>
>

-- 



The Cucumber logo is the intellectual property of Cucumber Ltd, a limited 
company registered in Scotland, number 456793.

UK Headquarters: Cucumber Ltd, Drumsyniebeg, Lochgoilhead, Cairndow, 
Argyll, PA24 8AN UK.

CONFIDENTIALITY NOTICE: The information in this e-mail is confidential and 
privileged; it is intended for use solely by the individual or entity named 
as the recipient hereof. Disclosure, copying, distribution, or use of the 
contents of this e-mail by persons other than the intended recipient is 
strictly prohibited and may violate applicable laws. If you have received 
this e-mail in error, please delete the original message and notify us by 
email immediately. Thank you. Cucumber Ltd.

mpkorstanje committed:
Add Cucumber for VSCode

While the documentation mentioned the existence of VS Code, it didn't
include the Cucumber for VSCode extension. A bit of an oversight.
